components/org.wso2.carbon.identity.sts.passive/pom.xml

```diff
@@ -96,12 +96,12 @@
         <dependency>
             <groupId>org.apache.cxf</groupId>
             <artifactId>cxf-core</artifactId>
-            <version>3.3.6</version>
+            <version>${cxf-bundle.version}</version>
```

Comment:
remove the version from the child pom

Reply:
Remove the versions from the child pom and added the dependencies to parent pom